core/src/main/java/nl/nn/adapterframework/http/HttpSender.java

```diff
@@ -327,13 +338,14 @@ protected FormBodyPart createMultipartBodypart(String name, InputStream is, Stri
 	}
 
 	protected FormBodyPart createMultipartBodypart(String name, InputStream is, String fileName, String contentType) {
+		log.debug(getLogPrefix()+"appending filepart ["+name+"] with value ["+is+"] fileName ["+fileName+"] and contentType ["+contentType+"]");
```

Comment:
Ik doe een complexe log.debug() altijd prefixen met if (log.isDebugEnabled()), t.b.v. performance in productie